include/gsl/gsl_assert

```diff
@@ -68,20 +68,54 @@
 
 namespace gsl
 {
+
 struct fail_fast : public std::runtime_error
 {
     explicit fail_fast(char const* const message) : std::runtime_error(message) {}
 };
-}
+namespace details
+{
+template <typename Exception>
+void throw_exception(Exception&& exception);
```

Comment:
As previously advertised, I'm dropping VS 2013 support as VS 2017 has been released. I'm using Casey's conveniently timed PR to fix CI to drive this (sorry to piggyback Casey :-)).

So feel free to use the attributes normally and I'll make sure this goes in after the CI changes.